### DISCUSSION TITLE
Geo-bypass option enabled? Audio quality selection?

### DISCUSSION THREAD

**Discussion by vasist2000** [JunkFood02/Seal#268] [Q&A] [answered]
Hi team, I'm new here and just wanted to know if the "geo-bypass" option of yt-dlp is enabled by default. I could not find the terminal command from your codebase. Please direct me toward the same. Also are we using the "best audio" option for audio downloads ?
I'm on seal v1.3.3-F-Droid. 
Thanks.

**Comment by JunkFood02** [comment, marked as answer]
`geo-bypass` is enabled by default in yt-dlp. `-x` means `bestaudio` when audio-only formats are available. So the answer for both questions: yes
